projects/addon-commerce/components/money/money-options.ts

```diff
@@ -1,21 +1,34 @@
-import {InjectionToken, ValueProvider} from '@angular/core';
+import {inject, InjectionToken, Provider} from '@angular/core';
 import {TuiCurrency} from '@taiga-ui/addon-commerce/enums';
-import {TuiCurrencyVariants, TuiMoneySignT} from '@taiga-ui/addon-commerce/types';
-import {TuiDecimalT} from '@taiga-ui/core';
+import {TuiCurrencyVariants} from '@taiga-ui/addon-commerce/types';
+import {tuiAssert} from '@taiga-ui/cdk';
+import {NumberFormatSettings, TUI_NUMBER_FORMAT, TuiDecimalT} from '@taiga-ui/core';
```

Comment:
why not `TuiNumberFormatSettings`

Reply:
fixed!

✅ deprecate old interface `NumberFormatSettings`
✅ replace all its usage by `TuiNumberFormatSettings`
✅ add migration for it